### ISSUE TITLE
How can I perform a multi-label training?

### ISSUE THREAD

**Issue by ShiyuZhangUST** [tensorflow/models#5191]
I want to train a model with 2 layers' output. For example, a person stand in front of a building, so that pixel belongs both to person and building. In my test, I add 2 softmax cross entropy for each layer. But the training result turned to be 2 layers with same color lump. The first layer's output is all class 1 and the second layer's output is all background.

Is the way I add cross entropy right?

    tf.losses.softmax_cross_entropy(
      one_hot_labels_layer1,
      tf.reshape(logits_layer1, shape = [-1, num_classes_layer1]),
      weights = mask_layer1,
      scope = '%s_%s' % (loss_scope, 'layer1'))

    tf.losses.softmax_cross_entropy(
      one_hot_labels_layer2,
      tf.reshape(logits_layer2, shape = [-1, num_classes_layer2]),
      weights = mask_layer2,
      scope = '%s_%s' % (loss_scope, 'layer2'))

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
